plugin/src/main/scala/io.github.jonas.paradox.material.theme/ParadoxMaterialThemePlugin.scala

```diff
@@ -23,24 +23,24 @@ object ParadoxMaterialThemePlugin extends AutoPlugin {
   )
 
   def paradoxMaterialThemeGlobalSettings: Seq[Setting[_]] = Def.settings(
-    version in paradoxMaterialTheme :=
+    paradoxMaterialTheme / version :=
```

Comment:
Do we have CI setup? I don't think this would work because this repo uses sbt 1.0.x to cross build the plugin.

Reply:
We don't have CI setup, that will be next PR and I prefer to setup SBT before setting up CI (I will also remove the cross version for sbt 0.13.x)